### PR TITLE
Limits update

### PR DESCRIPTION
**Pull request by daisyfaithauma**
- This PR is to update the limits page

### CODE REVIEW

**Comment by kathayl** [comment]
Cool - this page looks pretty much ready to go. Will just need to update with the metadata limit once we have that

File: src/content/docs/ai-gateway/reference/limits.mdx [hunk 1]
```diff
@@ -7,6 +7,17 @@ sidebar:
 
 import { Render } from "~/components";
 
-You can have a maximum of 10 gateways within your AI Gateway account.
+The following limits apply to gateway configurations, logs, and related features in Cloudflare's platform.
+
+| Feature                     | Limit                       |
+| --------------------------- | --------------------------- |
+| Maximum gateways            | 10 per account              |
+| Maximum logs stored         | 10 million logs per gateway |
```
Comment (kathayl) [inline src/content/docs/ai-gateway/reference/limits.mdx]: 1) can we clarify that this "maximum logs stored" limit is for paid users
2) can we add in a new line that is "maximum logs store" limit for free users is 100,000. And can link to pricing page

Comment (daisyfaithauma) [reply src/content/docs/ai-gateway/reference/limits.mdx]: Updated

**Comment by daisyfaithauma** [comment]
Metadata limit still not added.

**Review by Oxyjun — APPROVED** [review]

**Pull request merged**